### PR TITLE
Testing out new linter

### PR DESCRIPTION
**Pull request by esoergel**
testing linter, please ignore

### CODE REVIEW

File: corehq/apps/style/static/style/js/hq.helpers.js [hunk 1]
```diff
@@ -126,6 +126,10 @@ $.fn.enableButton = function () {
            .removeAttr('disabled');
 };
 
+if (nonexistant_variable == "doubleequals") {
```
Comment (dimagimon) [inline corehq/apps/style/static/style/js/hq.helpers.js]: 'nonexistant_variable' is not defined. (no-undef)
Expected '===' and instead saw '=='. (eqeqeq)

File: corehq/apps/style/static/style/js/hq.helpers.js [hunk 2]
```diff
@@ -126,6 +126,10 @@ $.fn.enableButton = function () {
...
+    f0ooo = "hello!"
```
Comment (dimagimon) [inline corehq/apps/style/static/style/js/hq.helpers.js]: 'f0ooo' is not defined. (no-undef)
Missing semicolon. (semi)

File: corehq/apps/style/static/style/js/hq.helpers.js [hunk 3]
```diff
@@ -136,3 +140,5 @@ $.fn.koApplyBindings = function (context) {
     }
     return ko.applyBindings(context, this.get(0));
 };
+
+console.log("does this error?");
```
Comment (dimagimon) [inline corehq/apps/style/static/style/js/hq.helpers.js]: Unexpected console statement. (no-console)

**Comment by benrudolph** [comment]
:eyes:

**Pull request closed without merging**